test/inductor/test_aot_inductor.py

```diff
@@ -63,9 +63,14 @@ def load(
             constraints=constraints,
         )
 
+        launcher = aot_inductor_launcher
+        is_cpu = any(x.device.type == "cpu" for x in example_inputs)
+        if is_cpu:
+            launcher = launcher.replace("false /*is_cpu*/", "true /*is_cpu*/")
```

Comment:
Err, using launcher.format(...) maybe?

Reply:
Heh, I wanted to avoid changing that launcher to something that needed to be formatted, just in case it was used elsewhere :-).  But sure.